edgekv/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "edgekv"
+version = "0.1.0"
```

Comment:
I think we want here and in db-checker version=1.10.0 to match the rest of the crates in the repository, what do you think?